great_expectations/datasource/fluent/schemas/PandasAzureBlobStorageDatasource.json

```diff
@@ -86,14 +86,14 @@
                 },
                 "batching_regex": {
                     "title": "Batching Regex",
+                    "default": ".*",
                     "type": "string",
                     "format": "regex"
                 }
             },
             "required": [
                 "name",
-                "type",
-                "batching_regex"
+                "type"
```

Comment:
`batching_regex` wasn't really a `required` field before because we always set it in the manually defined `add_<ASSET_TYPE>_method()`.

By using the pydantic field default, the jsonschema now correctly expresses these constraints and default value.

Reply:
🙇🏻